### ISSUE TITLE
I'm getting ERROR: Failed to resolve: Toasty Affected Modules: app

### ISSUE THREAD

**Issue by crimsonred501** [GrenderG/Toasty#113]
I'm getting ERROR: Failed to resolve: Toasty Affected Modules: app whenever I try to build.
Here is my gradle file:

```
buildscript {
    repositories {
        jcenter()
        google()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.2.1'
        classpath 'com.google.gms:google-services:3.2.0'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        mavenCentral()
        jcenter()
        maven {  url "https://maven.google.com" }
        maven {
            url "https://jitpack.io"
        }
        google()
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
```

**Comment by e4basil** [comment]
which version you are using?

try to use the latest version 

[check](https://jitpack.io/#GrenderG/Toasty)

**Comment by crimsonred501** [comment]
> which version you are using?
> 
> try to use the latest version
> 
> [check](https://jitpack.io/#GrenderG/Toasty)

Thanks. I fixed it by updating version from 1.2.5 to 1.4.2
